src/dstack/_internal/core/models/configurations.py

```diff
@@ -233,7 +233,10 @@ class ServiceConfiguration(BaseConfiguration):
         Field(description="Mapping of the model for the OpenAI-compatible endpoint"),
     ] = None
     auth: Annotated[bool, Field(description="Enable the authorization")] = True
-    replicas: Annotated[Range[int], Field(description="The range ")] = Range[int](min=1, max=1)
+    replicas: Annotated[
+        Union[conint(ge=1), constr(regex=r"^[0-9]+..[1-9][0-9]*$"), Range[int]],
```

Comment:
I'm afraid this breaks type checking. The type checker sees `replicas` as `Union[int, str, Range[int]]`, but we use it specifically as `Range[int]` ([1](https://github.com/dstackai/dstack/blob/04a62ba630fd775c05bbdeb0f9a7519a155bde46/src/dstack/_internal/server/services/gateways/autoscalers.py#L92), [2](https://github.com/dstackai/dstack/blob/04a62ba630fd775c05bbdeb0f9a7519a155bde46/src/dstack/_internal/server/background/tasks/process_runs.py#L133), [3](https://github.com/dstackai/dstack/blob/04a62ba630fd775c05bbdeb0f9a7519a155bde46/src/dstack/_internal/server/services/runs.py#L422), etc)